assessment.md

```diff
@@ -13,81 +13,512 @@
 * Calculating code coverage of a `pytest` test suite
+  * N = Did not utilize `pytest` for code coverage
+  * I = Test suite has low percent of coverage (30 and Below)
+  * A = Test suite has moderate percent of coverage (30 - 50)
+  * G = Test suite has an acceptable percent of coverage (50 - 80)
+  * E = Test suite has a high percent of coverage (80 - 100)
 * Reporting code coverage through an online provider
+  * N = Did not utilize online provider for code coverage
+  * I = Test suite has low percent of coverage (30 and Below)
+  * A = Test suite has moderate percent of coverage (30 - 50)
+  * G = Test suite has an acceptable percent of coverage (50 - 80)
+  * E = Test suite has a high percent of coverage (80 - 100)
 * Using docstrings to document a Python program
+  * N = Student did not use docstrings
+  * I = Students incorrectly attempted docstrings unidentifiable to compiler
+  * A = Student added docstrings, but it does not fully prescribe the functions/
+  methods effect as a command
+  * G = Student successfully added docstring with useful description, but is
+  lacking some proper syntax
+  * E = Student successfully added docstring with useful description and proper
+  syntax
 * Adopting and applying appropriate naming conventions
+  * N = Student used inappropriate naming styles that are neither prescriptive
+  or descriptive
+  * I = Student used inappropriate naming styles that are either not
+  prescriptive or descriptive
+  * A = Students used appropriate naming styles that are not entirely
+  conventional
+  * G = Student used appropriate naming styles, but is missing syntax used with
+  variables, function, and methods
+  * E = Student used appropriate naming styles with correct syntax used with
+  variables, functions, and methods
 * Using exception handling to create a robust Python program
+  * N = Student does use exception handling for the possibility of unexpected
+  user input
+  * I = Student introduces try and except statements but does not have any clause
+  * A = Student utilizes try and except statements or user defined exceptions,
+  however, clause does not function
+  * G = Student correctly codes exception handling, but clause doesn't fully execute
+  * E = Student correctly codes exception handling with clause being perfectly executed
 * Refactoring a Python program to improve its characteristics
+  * N = No refactoring done or changed functions completely
+  * I = Lack of documentation and clearly identifiable functions/code
+  * A = Some changes can be made but well documented
+  * G = Includes easy to understand functions and documentation
+  * E = Clean and efficient code that is well documented
 * Debugging Python programs using logging and other methods
+  * N = No debugging done
+  * I = Identified bugs in system but no fix
+  * A = Fixed some identified bugs using logging/other methods
+  * G = Fixed most identified bugs using logging/other methods
```

Comment:
There is no change from the A to the G category here. There should be a steady increase in the ability of each person.

Reply:
I would disagree here, as 'A' only requires some fixes, whereas 'G' requires most. Perhaps changing to 'few' and 'many' would be better?

Reply:
This has been addressed and changed